packages/core/docs/recipes/testing.md

```diff
@@ -0,0 +1,198 @@
+# Execute tests using Docker
+Usually, any project you develop requires Unit and Integrational tests.
+This recipe intended for Newbies, so if you are one of the Guru's that have their boilerplate, you can skip this document.
+
+Not every test you develop could be run on your local PC because they require additional Software such as Database Brokers or other services.
+
+These recipes do not attempt to become the Full Solution and try to show some simple appliances.
```

Comment:
These recipe is not intended to provide the full solution, just highlights the basic usecase